Source/WebCore/html/HTMLBodyElement.idl

```diff
@@ -35,6 +35,7 @@
     [WindowEventHandler] attribute EventHandler onload;
     [WindowEventHandler] attribute EventHandler onresize;
     [WindowEventHandler] attribute EventHandler onscroll;
+    [WindowEventHandler, EnabledBySetting=ScrollendEventEnabled] attribute EventHandler onscrollend;
```

Comment:
The second table on https://html.spec.whatwg.org/multipage/webappapis.html#event-handlers-on-elements,-document-objects,-and-window-objects says it shouldn't be part of this list.

This IDL file should probably link to: https://html.spec.whatwg.org/multipage/webappapis.html#window-reflecting-body-element-event-handler-set